web/thesauruses/javascript/ECMAScript 2020/operators.json

```diff
@@ -1,204 +1,220 @@
 {
   "meta": {
     "language": "javascript",
-    "language_version": "ECMAScript 2020",
     "language_name": "JavaScript",
-    "structure": "operators"
+    "structure": "operators",
+    "language_version": "ECMAScript 2020"
   },
   "concepts": {
     "addition": {
-      "code": "operand1 + operand2",
-      "name": "Addition operator"
+      "name": "Addition operator",
+      "code": "a + b"
```

Comment:
Why are you redoing these? They were correct before.

Reply:
I just shortened the expressions to make them more readable, but I can still replace them with names.

Reply:
That's fine. I'm not worried about short, but I guess if you're filling in others, it keeps it consistent.